### PR TITLE
Merging Chef12 changes

### PR DESCRIPTION
**Pull request by dexpetkovic**

### CODE REVIEW

**Review by ptqa — APPROVED** [review]

File: providers/jcr_node.rb [hunk 1]
```diff
@@ -60,16 +51,21 @@ def make_url(new_resource)
   case new_resource.type
   when 'file'
     unless check_node(url, new_resource.user, new_resource.password, new_resource.name) == new_resource.contents
-      fields = [
-        Curl::PostField.file(new_resource.name, new_resource.contents),
-        Curl::PostField.content("#{new_resource.name}@TypeHint", 'Binary')
-      ]
-      c = curl_form(url, new_resource.user, new_resource.password, fields)
-      if c.response_code == 200 || c.response_code == 201
+      # EH This doesn't work!! Fields need to be in right format, need to figure out what AEM api expects
```
Comment (madeddie) [inline providers/jcr_node.rb:54]: you might want to delete this en the comments below here :)

File: providers/jcr_node.rb [hunk 2]
```diff
@@ -81,12 +77,13 @@ def make_url(new_resource)
   url = make_url(new_resource)
   if check_node(url, new_resource.user, new_resource.password, new_resource.name)
     # If the node exists, delete it
-    fields = [Curl::PostField.content(':operation', 'delete')]
-    c = curl_form(url, new_resource.user, new_resource.password, fields)
-    if c.response_code == 200 || c.response_code == 201
+    # EH might not have to be multipart
```
Comment (madeddie) [inline providers/jcr_node.rb:80]: also this comment might confuse more than help, it was a TODO for myself, so, just throw it away :)

**Review by madeddie — APPROVED** [review]
apart from the ugly comments, i approve :)

File: providers/jcr_node.rb (hunk 1 above)
Comment (dexpetkovic) [reply providers/jcr_node.rb:54]: Will leave it as reminder to take care of it in future :)

**Pull request merged**